### PR TITLE
Updating version, hash and build number

### PR DESCRIPTION
**Pull request by ryanskeith**
# Changes
---
- updated version
- updated hash
- updated build number

# Notes
---
It seems 1.1.2 is already in main channel.  For almost everything except for s390x.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: https://github.com/google/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: a6533f45b1670b5d59b38a514d82b09c6fb70cc1050467220216335e873074e8
+  sha256: ac07840513072b7fcebda6e821068aa04889018f24e10e46181068fb214d7e56
 
 build:
-  number: 2
+  number: 1
```
Comment (katietz) [inline recipe/meta.yaml]: as you raised version, the build number should be reset for it to zero

Comment (katietz) [reply recipe/meta.yaml]: additionally the run_export part here should go away

**Review by katietz — CHANGES_REQUESTED** [review]

**Comment by skupr-anaconda** [comment]
- Missing `license_url: https://github.com/google/crc32c/blob/main/LICENSE`
- Missing `dev_url: https://github.com/google/crc32c`
- Missing `doc_url: https://github.com/google/crc32c/blob/main/README.md`
- Missing a description: `CRC32C is specified as the CRC that uses the iSCSI polynomial in RFC 3720. The polynomial was introduced by G. Castagnoli, S. Braeuer and M. Herrmann. CRC32C is used in software such as Btrfs, ext4, Ceph and leveldb.`

File: recipe/meta.yaml (hunk 1 above)
Comment (ryanskeith) [reply recipe/meta.yaml]: There already exists a build 0 for this so I think build 1 is appropriate.

**Review by varlackc — COMMENTED** [review]
Other than the missing components and the issues mentioned, everything else seems in order. As soon as the requested changes are added we should be good to approve the PR.

File: recipe/meta.yaml (hunk 1 above)
Comment (ryanskeith) [reply recipe/meta.yaml]: I was horribly mistaken.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**